### ISSUE TITLE
How can I install a local instance?

### ISSUE THREAD

**Issue by moshfeu** [tma02/baritone#9]
Thanks for this cool plugin!

After I cloned this project how can I run it?

I'm asking about cloned project because I want to add functionality to it.

Thanks!

**Comment by tma02** [comment]
Hi!
I've just updated the readme with instructions on running the project from source.
Let me know if you still need assistance.

**Comment by moshfeu** [comment]
Thanks! I followed the instructions and: the install process never and, just write to console:

`electron-prebuilt@0.37.8 postinstall /Users/moshef/Documents/Github/baritone/node_modules/electron-prebuilt
> node install.js


... and 1 more`

When I try `node test` I got error include `no such file or directory, open '/Users/moshef/Documents/Github/baritone/node_modules/electron-prebuilt/path.txt'`.

When I try to run again `node install.js` in the relevant folder it doesn't help and the file `path.txt` didn't created.

Any thoughts? 
Thanks!
